common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -248,6 +251,8 @@ object LoggingMarkers {
   val INVOKER_ACTIVATION = LogMarkerToken(invoker, activation, start)
   def INVOKER_DOCKER_CMD(cmd: String) = LogMarkerToken(invoker, s"docker.$cmd", start)
   def INVOKER_RUNC_CMD(cmd: String) = LogMarkerToken(invoker, s"runc.$cmd", start)
+  def INVOKER_CONTAINER_START(actionName: String, namespaceName: String, containerState: String) =
+    LogMarkerToken(invoker, s"container_start_${containerState}_${namespaceName}_$actionName", count)
```

Comment:
will this allow you to count containers started for a namespace vs ones specific to an action? ie can you aggregate ignoring the action?

Reply:
@rabbah Grafana Yes. Grafana allows to aggregate values on any 'regexable' - sub-string of the metric name

Reply:
same comment re param typing.

Reply:
Same as below, would be `EntityName`. Is that much more valuable?